src/templates/view-edit-toolbar.html

```
@@ -155,190 +155,201 @@ <h5 class='modal-title' id='procurementRequestModalLabel'>{{ 'Create procurement
+        <a title='{{ 'Save'|trans }}' data-action='update-entity-body' aria-label='{{ 'Save'|trans }}' class='btn btn-primary mr-2'>
+          <i class='fas fa-save'></i>
         </a>
-        <button type='button' class='btn btn-transparent mr-2' aria-label='{{ 'Send an email to users who have booked this resource within the 4 last & upcoming months.'|trans }}' title='{{ 'Send an email to users who have booked this resource within the 4 last & upcoming months.'|trans }}' data-action='toggle-modal' data-target='sendBookingsEmailModal'>
-          <i class='fas fa-fw fa-envelope'></i>
-        </button>
-      {% else %}
-        <button type='button' class='btn mr-2 btn-transparent' aria-label='{{ 'Modify booking parameters'|trans }}' title='{{ 'Modify booking parameters'|trans }}' data-action='toggle-modal' data-target='bookingParamsModal'>
-          <i class='fas fa-calendar-plus color-disabled fa-fw'></i>
-        </button>
-      {% endif %}
+      <a title='{{ 'Save and go back'|trans }}' aria-label='{{ 'Save ang go back'|trans }}' href='?mode=view&amp;id={{ Entity.id }}' data-redirect='view' class='btn btn-transparent mr-2'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)